internal/controller/driver_controller.go

```diff
@@ -226,6 +226,10 @@ func (r *DriverReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 func (r *driverReconcile) reconcile() error {
 	// Load the driver desired state based on driver resource, operator config resource and default values.
 	if err := r.LoadAndValidateDesiredState(); err != nil {
+		if client.IgnoreNotFound(err) == nil {
+			r.log.Info("Resources for reconciling driver is not found. Object(s) might have been deleted.")
```

Comment:
Let's be more specific in the log message:
```suggestion
			r.log.Info("Driver resource does not exist anymore, skipping reconcile.")
```